src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ConditionSpecificLogic.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
```

Comment:
this might be better to be initial data specific, will see what happens when adding the update aspect of the story